cluster/kube-util.sh

```diff
@@ -38,27 +38,34 @@ fi
 # Should NOT be called within the global scope, unless setting the desired global zone vars
 # This function is currently NOT USED in the global scope
 function set-federation-zone-vars {
```

Comment:
I wrote most of this file, so I felt compelled to fix the indent problem on the entire function.